microcosm_postgres/operations.py

```diff
@@ -93,7 +92,12 @@ def recreate_all(graph):
     connection = graph.postgres.connect()
     transaction = connection.begin()
     for table in reversed(_metadata.sorted_tables):
-        connection.execute(table.delete())
+        # Occasionally SqlAlchemy can't resolve the order in which to truncate records, due to cyclic
+        # Foreign Key constraints. We can use this flag to ensure truncation cascades between records.
+        if cascade_truncation:
+            connection.execute(f"TRUNCATE TABLE \"{table}\" CASCADE")
```

Comment:
I think you can use `public.{table}` for a similar effect, if you want to avoid namespace clashes.